applications/asset_tracker/README.rst

```diff
@@ -77,28 +85,41 @@ LED 3 and LED 4:
 
 All LEDs (1-4):
     * Blinking in groups of two (LED 1 and 3, LED 2 and 4): Recoverable error in the BSD library.
-    * Blinking in cross pattern (LED 1 and 4, LED 2 and 3): Communication error with the nRF Cloud.
+    * Blinking in cross pattern (LED 1 and 4, LED 2 and 3): Communication error with nRF Cloud.
+
+On the Thingy:91, the application state is indicated by a single RGB LED as follows:
+
+.. csv-table:: Thingy:91 LED states
+   :header: "Color", "State"
+   "White", "Connecting to network"
+   "Cyan", "Connecting to nRF Cloud"
+   "Yellow", "Waiting for user association"
+   "Blue", "Connected, sending environment data"
+   "Purple", "Searching for GPS"
+   "Green", "Connected, sending GPS and environment data"
+   "Red", "Error"
 
 .. _power_opt:
 
 Power optimization
 ******************
 
-The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application.
+The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application. These settings are currently only supported on the nRF9160 DK.
 
 Demo mode
 	This is the default setting.
-	In this mode, the device sends GPS data every 2 seconds.
+	In this mode, the device maintains a continuous cellular link.
 	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=n``.
 
 Request eDRX mode
-	In this mode, the device sends GPS data every 2 minutes.
+	In this mode, the device requests the eDRX feature from the cellular network to save power.
 	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
-	set Switch 2 to ON.
+	set Switch 2 to N.C.
 
 Request Power Saving Mode (PSM)
-	To enable PSM, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
-	set Switch 2 to OFF.
+	In this mode, the device requests the PSM feature from the cellular network to save power.
+	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
+	set Switch 2 to GND.
```

Comment:
Same note as for eDRX may be relevant here. Depends on how detailed we should be on explaining these features in the application docs, though.

Reply:
Added to section intro